.github/workflows/manual-integ.yml

```diff
@@ -5,10 +5,14 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - Python_3.8
-          - Python_3.9
+          - "3.9"
+          - "3.10"
+          - "3.11"
```

Comment:
Why is 3.8 not included here?

Reply:
Thanks for pointing this out. I was thinking we should limit the number of python versions the integration tests cover to avoid an unwieldy number of integration tests. In this PR, I've excluded python 3.8 so that the integration tests cover only the latest 3 major python versions. When diving into improving integration tests in the future, we could make this dynamic to where it reads the last three python versions in the list in setup.py. 

As we add more support for python versions in the future, this window shifts. For example, if we add 3.12 in the future, the window will automatically shift to run integration tests on 3.10, 3.11, and 3.12.

I'm open to further discussing this or even keeping 3.8. Let me know your thoughts!

Reply:
To simplify things, I'll keep 3.8 for this PR. I'll create an issue to create discussion on how the community would like to go forth with this integ testing python versions.

Reply:
Created an issue regarding this discussion: https://github.com/opensearch-project/opensearch-benchmark/issues/228